connector_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Snowflake Computing Inc. All right reserved.
+
+package gosnowflake
+
+import (
+	"context"
+	"database/sql/driver"
+	"reflect"
+	"testing"
+)
+
+type snowflakeMockDriver struct {
```

Comment:
Can we call this `noopTestDriver` or something similar? We may be introducing a mock driver in a future PR and it could be confused with this.

Reply:
Yes

Reply:
renamed